recipes/MobileSAM/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "mobilesam" %}
+{% set version = "1.0.0.20231030.12d80d4e32b277de299130d8ce28cc949fb54b6c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/ChaoningZhang/MobileSAM.git
+  git_rev: 12d80d4e32b277de299130d8ce28cc949fb54b6c
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install . -vv
```

```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

the example suggests